mods/cnc/maps/gdi05c/map.yaml

```diff
@@ -0,0 +1,768 @@
+MapFormat: 11
+
+RequiresMod: cnc
+
+Title: 05c: Repair the GDI base
+
+Author: Westwood Studios
+
+Tileset: TEMPERAT
+
+MapSize: 64,64
+
+Bounds: 9,1,54,45
+
+Visibility: MissionSelector
+
+Categories: Campaign
+
+LockPreview: True
+
+Players:
+	PlayerReference@Nod:
+		Name: Nod
+		Faction: nod
+		Color: FE1100
+		Enemies: GDI, AbandonedBase
+		Bot: campaign
+	PlayerReference@GDI:
+		Name: GDI
+		Playable: True
+		AllowBots: False
+		Required: True
+		LockFaction: True
+		Faction: gdi
+		LockColor: True
+		Color: F5D378
+		LockSpawn: True
+		LockTeam: True
+		Enemies: Nod
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: gdi
+		Color: F5D378
+	PlayerReference@AbandonedBase:
+		Name: AbandonedBase
+		NonCombatant: True
+		Faction: gdi
+		Color: F5D378
+		Enemies: Nod
+		Bot: campaign
+
+Actors:
+	Actor0: sbag
+		Location: 62,34
+		Owner: Neutral
```

Comment:
It's a pain but walls need to be assigned a correct owner so the enemy player can target them.